buddyboss/buddyboss-platform
# Fixed injecting script/image in the document through URL

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2320 .

### How to test the changes in this Pull Request:

1. To see the issue please browser the following URL: https://dev-mahedi.flywheelsites.com/?bp_search=1&view=content&s=1%22%27%3E%3Cimg%20src=%22https://www.w3schools.com/html/pic_trulli.jpg%22%20alt=%22Italian%20Trulli%22%3E
2. Also please see the screenshot: https://prnt.sc/117ocpp
3. HTML should be escaped after this fix.

### Proof Screenshots or Video

https://www.loom.com/share/b192fde35fe7415aa4354b7a52f3e105

### Other information:




### Changelog entry

Escaped data and secured output
